charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Project Name
+HCA Data Processing Service
+
+## Description
+
+The data processing service is a centralized space to perform all transformative functionality to scientific data ingested into the Human Cell Atlas Data Coordination Platform (HCA DCP).
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__On-boarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay.  
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+
+## Objectives
+
+The data processing service aims to develop scientific analysis pipelines and the infrastructure to run them reliably and at-scale for all analyses on all scientific data types in the Human Cell Atlas Data Coordination Platform. The team working on the service will also operate the infrastructure. Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community.
+
+## In-scope
+
+* Scoping or review of scoping for new pipelines.  
+* Creation of all standardized pipelines for scientific data.  
+* Review, on-boarding, and potential creation of community pipelines.  
```

Comment:
To enable third parties to create community pipelines, there needs to be a modest amount of support infra (minimally documentation; maximally tooling or support).   Suggest that at least the minimum enabling "stuff" be included.   

Proposed addition:   "Tools which support the creation of community pipelines - may include software, documentation or support."

("may" indicates that it is optional - just leaving open the possibility that the team decides any or all of these are needed)

Reply:
Great point, Bruce - I think Tim is planning to make a separate charter for Unity, the service we are creating to enable more community creation of pipelines. I do think that documentation / support around the pipelines we run can fall under Data Processing Service charter, though. Thoughts?

Reply:
I'm in favor of having it in this charter, as it makes it clear that the community needs to be enabled by the team (within reason - obviously it is not a replacement for Unity)

Reply:
Thanks, this language is placed in the HCA Data Processing Pipelines charter, new version coming shortly.

Reply:
Tend to agree with Bruce here and think that some sort of pointer would be helpful. Given that this charter is a overarching statement and relates to more specific charters (eg pipelines, unity) indicating that support for enabling community contribution is a key feature of the top level goals.